### PR TITLE
1st Art Pass

### PR DESCRIPTION
**Pull request by KyleGospo**

### CODE REVIEW

File: usr/share/pixmaps/faces/bicycle.jpg
Comment (castrojo) [inline usr/share/pixmaps/faces/bicycle.jpg:1]: This one isn't done yet so I'll have a followup.

File: usr/share/pixmaps/faces/book.jpg
Comment (castrojo) [inline usr/share/pixmaps/faces/book.jpg:1]: This one is also draft

File: usr/share/pixmaps/faces/plane.jpg
Comment (KyleGospo) [inline usr/share/pixmaps/faces/plane.jpg:1]: draft

File: usr/share/pixmaps/faces/surfer.jpg
Comment (KyleGospo) [inline usr/share/pixmaps/faces/surfer.jpg:1]: draft

**Review by castrojo — APPROVED** [review]

**Review by castrojo — APPROVED** [review]

**Pull request merged**